src/components/DiffView/index.tsx

```diff
@@ -0,0 +1,28 @@
+/// <reference path="index.d.ts"/>
+
+import * as React from 'react';
+import { Diff, DiffInfo, DiffProps, parseDiff } from 'react-diff-view';
+
+type Props = {
+  diff: string;
+  viewType: DiffProps['viewType'];
+};
+
+class DiffView extends React.Component<Props> {
+  static defaultProps = {
+    viewType: 'unified',
+  };
+
+  render = () => {
+    const { diff, viewType } = this.props;
+    const files = parseDiff(diff);
+
+    const renderFile = ({ hunks, type }: DiffInfo, index: number) => (
+      <Diff diffType={type} hunks={hunks} key={index} viewType={viewType} />
+    );
+
+    return <div>{files.map(renderFile)}</div>;
```

Comment:
It would be nice to have a test for multiple `<Diff>` hunks.

Reply:
The problem I am having is that I cannot get `parseDiff` to generate output with multiple files. That's actually why I wrote the code without support for multiple files in the first place. Even using the sample diff at https://github.com/otakustay/react-diff-view/blob/v2.0.0-alpha.1/src/__test__/cases.js#L16, when I pass that into `parseDiff` it only extracts info for the first file.

This might be a bug with `parseDiff` that we're bound to come across when we try to use this library with actual data from our API, but I don't think it's worth trying to solve at this point. As I think I've already mentioned, this entire component will probably be replaced when we start trying to use this library to do actual things with our actual data. I only implemented it as a test to prove that we can at least use the `Diff` component to render something.

So while I agree that it makes sense to have code here that deals with possible multiple files, I don't think we should spend a lot of effort creating a test for it when the supporting code (`parseDiff`) may itself have a bug. Does this make sense?

Reply:
Huh, weird. If you get a minute, it might be helpful to file a bug about it.

Reply:
Good idea, done at https://github.com/otakustay/react-diff-view/issues/33